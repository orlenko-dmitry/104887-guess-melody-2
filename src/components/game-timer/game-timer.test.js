import React from 'react';
import {render} from '@testing-library/react';

import GameTimer from '../game-timer/game-timer.jsx';

it(`GameTimer renders correctly`, () => {
  const {container} = render(<GameTimer gameTime={300} currentTime={0} />);

  expect(container).toMatchSnapshot();
});
