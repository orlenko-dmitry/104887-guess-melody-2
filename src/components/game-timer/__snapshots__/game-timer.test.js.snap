// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameTimer renders correctly 1`] = `
<div>
  <div
    class="timer__value"
    xmlns="http://www.w3.org/1999/xhtml"
  >
    <span
      class="timer__mins"
    >
      05
    </span>
    <span
      class="timer__dots"
    >
      :
    </span>
    <span
      class="timer__secs"
    >
      00
    </span>
  </div>
</div>
`;
